### DISCUSSION TITLE
Обратная связь по CLI скрипту для добавления компонентов

### DISCUSSION THREAD

**Discussion by aak-lear** [quarx-ui/quarx#82] [Ideas]
Для упрощения работы с репозиторием дизайн-системы мы создали CLI-скрипт `new`, который позволяет генерировать файловую структуру новых компонентов, а также тестов и документации к ним.

В этой ветке можно оставить свои отзывы, пожелания и предложения по работе скрипта. Достаточно просто поделиться своими мыслями в комментариях 😉

**Comment by maximzadorkin** [comment]
Оставлю первый комментарий :)

Добавить возможность генерации pw тестов в e2e.

**Comment by maximzadorkin** [comment]
Добавить в шаблон тестов testStyleParams вместо текущей версии

**Comment by maximzadorkin** [comment]
Добавить  BaseProps для types.ts, чтобы не писать каждый раз ref и className

**Comment by aak-lear** [comment]
Насколько я понимаю, вот этот промах тоже связан с автогенерацией?
https://github.com/quarx-ui/quarx/pull/100#discussion_r1011227412

**Reply by maximzadorkin** [reply]
Возможно, с пробелом есть проблемы. Но прямо не в своем списке точно не мог оказаться
